src/app/contact/contact.component.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
need to create components folder and all the components you need to add in that folder